04.class/memo.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+import { MemoRepository } from "./memo_repository.js";
+
+const memoRepo = new MemoRepository();
+
+const option = process.argv[2];
+if (option === "-l") {
+  memoRepo.showList();
+} else if (option === "-r") {
+  memoRepo.showFullContent();
+} else if (option === "-d") {
+  memoRepo.delete();
+} else {
+  memoRepo.add();
+}
```

Comment:
変数のセットアップと実際の処理という観点で分けるなら、変数の定義順や空行の位置は以下のようになっていたほうがよいと思います。現在の空行の位置だと、どういうまとまりで分けられているのかいまいち分からないです。

```suggestion
const option = process.argv[2];
const memoRepo = new MemoRepository();

if (option === "-l") {
  memoRepo.showList();
} else if (option === "-r") {
  memoRepo.showFullContent();
} else if (option === "-d") {
  memoRepo.delete();
} else {
  memoRepo.add();
}
```

Reply:
修正しました。